### ISSUE TITLE
deploy demo app?

### ISSUE THREAD

**Issue by dcsan** [GraphemeNFT/rarible-scaffold#35]
for the judges we might need to deploy a version of this app?
that will be some work, getting all the wallets figured out.

since its all client /static, maybe we can just stick onto netlify? 

or I can setup a subdomain and a deploy location if we need anything more, but would need a bit of time.
not sure if it's needed, but always helps prove it's "real"

**Comment by dcsan** [comment]
first step is to use ropsten network, so items show up on rarible/opensea, but still demo "running" locally.

**Comment by keepRunning** [comment]
We can also use github action + pages and ropsten

**Comment by dcsan** [comment]
that sounds more complex to setup than just a static server with files. 
what would we need the github actions for? automated deployment?

**Comment by keepRunning** [comment]
yeah, might be too late for CI now. We do need a functional demo site.

I the priority as 
1. Deck for submission( by sunday night)
2. Somewhat functional app for demo ( could extend to  monday before they start judging)
3. Implement all innovative ideas ( a week or more)

**Comment by tomosaigon** [comment]
GitHub Pages is just static files. I don't want to be dependent on more companies so let's use GitHub Pages and instead of CI we manually build and push the files to the Pages repo.

**Comment by dcsan** [comment]
do you think this is critical? could be good if judges take the time.
